guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to different needs. These configurations include **Federated mints**, **Federated single guardian mints**, and **Solo mints (Unfederated mints)**, each with its strengths and trade-offs.
+
+## Role of guardians
+Guardians are participants who collaboratively manage a federations operations. They secure the federation's funds using multi-sig wallets, validate payments through consensus, and control ecash issuance and redemption.
+
+## Role of gateways
+
+{% include picture.html
```

Comment:
Notes from offline discussion: The graphic here could convey so much more information. I'd include the following:

* A gateway that serves multiple federations
* A gateway that serves a single federation
* A gateway with a lightning channel directly connected to another gateway
* A federation that is served by multiple gateways

We should also differentiate between real lightning channels and gateway<->federation connections